test/Atmos/EDMF/edmf_kernels.jl

```diff
@@ -742,24 +768,22 @@ function turbconv_normal_boundary_flux_second_order!(
     diff⁺::Vars,
     hyperdiff⁺::Vars,
     aux⁺::Vars,
-    bctype,
     t,
     _...,
 ) where {FT}
     turbconv = m.turbconv
-    N = n_updrafts(turbconv)
+    N_up = n_updrafts(turbconv)
     up_flx = fluxᵀn.turbconv.updraft
     en_flx = fluxᵀn.turbconv.environment
-    if bctype == 2 # top
-        @unroll_map(N_up) do i
-            up_flx[i].ρaw = -n⁻ * FT(0)
-            up_flx[i].ρa = -n⁻ * FT(0)
-            up_flx[i].ρaθ_liq = -n⁻ * FT(0)
-            up_flx[i].ρaq_tot = -n⁻ * FT(0)
-        end
-        en_flx.∇tke = -n⁻ * FT(0)
-        en_flx.∇e_int_cv = -n⁻ * FT(0)
-        en_flx.∇q_tot_cv = -n⁻ * FT(0)
-        en_flx.∇e_int_q_tot_cv = -n⁻ * FT(0)
-    end
+    # @unroll_map(N_up) do i
+    #     up_flx[i].ρaw = -n⁻ * FT(0)
+    #     up_flx[i].ρa = -n⁻ * FT(0)
+    #     up_flx[i].ρaθ_liq = -n⁻ * FT(0)
+    #     up_flx[i].ρaq_tot = -n⁻ * FT(0)
+    # end
+    # en_flx.∇tke = -n⁻ * FT(0)
+    # en_flx.∇e_int_cv = -n⁻ * FT(0)
+    # en_flx.∇q_tot_cv = -n⁻ * FT(0)
+    # en_flx.∇e_int_q_tot_cv = -n⁻ * FT(0)
```

Comment:
@charleskawczynski This code didn't work (previously it wasn't being called, as the top boundary condition didn't use it).

Reply:
Ah, good to know, thank you. Feel free to remove it for now, I'll open an issue about adding top BCs back in. cc @yairchn